admin/js/editor.js

```diff
@@ -44,17 +44,18 @@ wp.domReady(() => {
     ]),
   );
 
-  ['core/group'].forEach(block => {
-    registerBlockStyle(block, {
-      name: 'space-evenly',
-      label: __('Space evenly', 'planet4-blocks-backend'),
-    });
+  registerBlockStyle('core/group', {
```

Comment:
I'm not sure about this change since we'll potentially register this style to other blocks.
Does it make sense?